README.md

```diff
@@ -205,6 +207,35 @@ then we support Opera 12.1x, 19.x and 20.x but not Opera 15.x through 18.x.
 
 Cross-browser testing is sponsored by [SauceLabs](https://saucelabs.com).
 
+## Contributors
+
+This project exists thanks to all the people who contribute. <img src="https://opencollective.com/opal/contributors.svg?width=890&button=false" />
```

Comment:
I think it would make sense to make this image a link to the contributors page, also considering the fact that on GH it's a link to the image by default.

```suggestion
This project exists thanks to all the people who contribute. [![contributors](https://opencollective.com/opal/contributors.svg?width=890&button=false")](https://github.com/opal/opal/graphs/contributors)
```
```